src/multibody/geometry.hpp

```diff
@@ -260,10 +272,11 @@ namespace se3
     /// \brief Compute the collision status between two collision objects of a given collision pair.
     ///
     /// \param[in] pair The collsion pair.
+    /// \param[out] result where the collision result is stored.
     ///
     /// \return Return true is the collision objects are colliding.
     ///
-    CollisionResult computeCollision(const CollisionPair & pair) const;
+    bool computeCollision(const CollisionPair & pair, fcl::CollisionResult& result) const;
```

Comment:
This is mostly used internally like `computeCollision(colPairIndex, collisionResults[colPairIndex])`. Do you see any disadvantage in this ?
I did this way because:
1. I don't think it hurts,
2. it was useful somewhere when I wrote it but I forgot where... (yes, I know it is a bad reason)